README.md

```diff
@@ -12,7 +12,7 @@ This release provides the following features:
 ## Limitations and workarounds

```

Comment:
Could be squashed with previous commit.

Reply:
Ok, I will squash README.md and COPYING.MIT into 1 commit.